### ISSUE TITLE
No email notification when admin edits fixture

### ISSUE THREAD

**Issue by axunonb** [axuno/Volleyball-League#32]
When an admin edit a fixture, there are no notification emails generated after saving.
Submitted data is however stored in the database.

Log:
```
BackgroundTask.BackgroundQueue|Exception when executing a IBackgroundTask.  System.InvalidOperationException: Sequence contains no matching element
```

**Comment by axunonb** [comment]
Same is the case, when an admin edits a result

**Comment by axunonb** [comment]
Resolved with #78
